hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/MetricsHBaseServer.java

```diff
@@ -116,6 +122,12 @@ public void exception(Throwable throwable) {
         source.multiActionTooLargeException();
       } else if (throwable instanceof CallQueueTooBigException) {
         source.callQueueTooBigException();
+      } else if (throwable instanceof QuotaExceededException) {
+        source.quotaExceededException();
+      } else if (throwable instanceof RpcThrottlingException) {
+        source.rpcThrottlingException();
+      } else if (LOG.isDebugEnabled()) {
+        LOG.debug("Unknown exception type: " + throwable);
```

Comment:
Pass the exception as a parameter,

` LOG.debug("Unknown exception type: ", throwable);`

Reply:
Ugh sorry, I moved too quickly.

Reply:
Fixed